hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This patch removes a now-incorrect internal assertion about numpy's typing after recent numpy changes.
```

Comment:
might want to mention that this was in a nightly?

Reply:
I thought doing so would be confusing once the nightly was properly released, but on reflection you're right it's better to be explicit.